src/lightning_app/cli/lightning_cli.py

```diff
@@ -49,10 +51,89 @@ def main():
 @click.group()
 @click.version_option(ver)
 def _main():
-    register_all_external_components()
+    # register_all_external_components()
```

Comment:
Why this change? I should belong to another PR.

Reply:
You are absolutely right. Mea culpa here 😓